modules/spring-data-2.2/src/test/java/org/apache/ignite/testsuites/IgniteSpringData2TestSuite.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.testsuites;
+
+import org.apache.ignite.springdata.IgniteSpringDataCrudSelfTest;
+import org.apache.ignite.springdata.IgniteSpringDataQueriesSelfTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * Ignite Spring Data 2.2 test suite.
+ */
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    IgniteSpringDataCrudSelfTest.class,
+    IgniteSpringDataQueriesSelfTest.class
+})
+public class IgniteSpringData2TestSuite {
```

Comment:
Can you please rename this suite to IgniteSpringData22TestSuite or something like that? Per our TC we can't run suites if they have same class name.

Please also rebase on top of fresh master if possible.

Reply:
As I understand you already merged PR to master. Should I make this change?

Reply:
I think it is no longer relevant :) took me three commits to figure it out, though.